ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNTA4Mzk4MTMzLjE3NDYzMTI5NzYifSwiYW5vbklkIjoiMWU1ODM0NmIzODI4YWJhZCJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5pdC9wYXNzZW5nZXJjYXJzL21vZGVscy9zdXYvZy1jbGFzcy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI2MyIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjMwNjA5MzUxNzI4NTUwMTQiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55640f6a975a <unknown>
#1 0x55640f15c4b0 <unknown>
#2 0x55640f142d86 <unknown>
#3 0x55640f14169a <unknown>
#4 0x55640f141dff <unknown>
#5 0x55640f141d54 <unknown>
#6 0x55640f16069f <unknown>
#7 0x55640f20592a <unknown>
#8 0x55640f1d3592 <unknown>
#9 0x55640f1f95e0 <unknown>
#10 0x55640f1d3363 <unknown>
#11 0x55640f19fd63 <unknown>
#12 0x55640f1a09c1 <unknown>
#13 0x55640f66ea6b <unknown>
#14 0x55640f672951 <unknown>
#15 0x55640f656b62 <unknown>
#16 0x55640f6734c4 <unknown>
#17 0x55640f63b13f <unknown>
#18 0x55640f6976f8 <unknown>
#19 0x55640f6978d6 <unknown>
#20 0x55640f6a85a6 <unknown>
#21 0x7f253789caa4 <unknown>
#22 0x7f2537929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyA0NjUyIiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJpdF9JVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIn0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI1MDgzOTgxMzMuMTc0NjMxMjk3NiJ9LCJhbm9uSWQiOiIxZTU4MzQ2YjM4MjhhYmFkIn0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6Mn0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiI0NjUyIiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyI0NjUyIl19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMzc4OTQwNzcxMzU5MzIzIn0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55640f6a975a <unknown>
#1 0x55640f15c4b0 <unknown>
#2 0x55640f142d86 <unknown>
#3 0x55640f14169a <unknown>
#4 0x55640f141dff <unknown>
#5 0x55640f141d54 <unknown>
#6 0x55640f16069f <unknown>
#7 0x55640f20592a <unknown>
#8 0x55640f1d3592 <unknown>
#9 0x55640f1f95e0 <unknown>
#10 0x55640f1d3363 <unknown>
#11 0x55640f19fd63 <unknown>
#12 0x55640f1a09c1 <unknown>
#13 0x55640f66ea6b <unknown>
#14 0x55640f672951 <unknown>
#15 0x55640f656b62 <unknown>
#16 0x55640f6734c4 <unknown>
#17 0x55640f63b13f <unknown>
#18 0x55640f6976f8 <unknown>
#19 0x55640f6978d6 <unknown>
#20 0x55640f6a85a6 <unknown>
#21 0x7f253789caa4 <unknown>
#22 0x7f2537929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjUwODM5ODEzMy4xNzQ2MzEyOTc2In0sImFub25JZCI6IjFlNTgzNDZiMzgyOGFiYWQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouaXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2ctY2xhc3Mvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNjMiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoyfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiIwMDQ2NDMwMDMyNzQwNjM3NTEiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55640f6a975a <unknown>
#1 0x55640f15c4b0 <unknown>
#2 0x55640f142d86 <unknown>
#3 0x55640f14169a <unknown>
#4 0x55640f141dff <unknown>
#5 0x55640f141d54 <unknown>
#6 0x55640f16069f <unknown>
#7 0x55640f20592a <unknown>
#8 0x55640f1d3592 <unknown>
#9 0x55640f1f95e0 <unknown>
#10 0x55640f1d3363 <unknown>
#11 0x55640f19fd63 <unknown>
#12 0x55640f1a09c1 <unknown>
#13 0x55640f66ea6b <unknown>
#14 0x55640f672951 <unknown>
#15 0x55640f656b62 <unknown>
#16 0x55640f6734c4 <unknown>
#17 0x55640f63b13f <unknown>
#18 0x55640f6976f8 <unknown>
#19 0x55640f6978d6 <unknown>
#20 0x55640f6a85a6 <unknown>
#21 0x7f253789caa4 <unknown>
#22 0x7f2537929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFBpUGFnZSI6eyJ2ZWhpY2xlUEkiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ2ZWhpY2xlQ29kZSI6Inc0NjUiLCJ2ZWhpY2xlQk0iOiI0NjUyIiwidmVoaWNsZVBJSGlzdG9yeSI6IjQ2NTIiLCJ0aW1lc3RhbXAiOjE3NDYzMTI5ODczODEsImV4cGlyYXRpb24iOjEyMH19fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI1MDgzOTgxMzMuMTc0NjMxMjk3NiJ9LCJhbm9uSWQiOiIxZTU4MzQ2YjM4MjhhYmFkIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI0MDcyMTg5MzMwMTUyNzUzNCIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55640f6a975a <unknown>
#1 0x55640f15c4b0 <unknown>
#2 0x55640f142d86 <unknown>
#3 0x55640f14169a <unknown>
#4 0x55640f141dff <unknown>
#5 0x55640f141d54 <unknown>
#6 0x55640f16069f <unknown>
#7 0x55640f20592a <unknown>
#8 0x55640f1d3592 <unknown>
#9 0x55640f1f95e0 <unknown>
#10 0x55640f1d3363 <unknown>
#11 0x55640f19fd63 <unknown>
#12 0x55640f1a09c1 <unknown>
#13 0x55640f66ea6b <unknown>
#14 0x55640f672951 <unknown>
#15 0x55640f656b62 <unknown>
#16 0x55640f6734c4 <unknown>
#17 0x55640f63b13f <unknown>
#18 0x55640f6976f8 <unknown>
#19 0x55640f6978d6 <unknown>
#20 0x55640f6a85a6 <unknown>
#21 0x7f253789caa4 <unknown>
#22 0x7f2537929c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_italy_prod?event=eyJhY3Rpb24iOiIxNSBTZWNvbmRzIG9uIFBJIFBhZ2UgLSA0NjUyIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI1MDgzOTgxMzMuMTc0NjMxMjk3NiJ9LCJhbm9uSWQiOiIxZTU4MzQ2YjM4MjhhYmFkIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lml0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nLWNsYXNzL292ZXJ2aWV3Lmh0bWwiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjYzIiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNTgxMjQyNDMxMDg5MDYxNyIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55640f6a975a <unknown>
#1 0x55640f15c4b0 <unknown>
#2 0x55640f142d86 <unknown>
#3 0x55640f14169a <unknown>
#4 0x55640f141dff <unknown>
#5 0x55640f141d54 <unknown>
#6 0x55640f16069f <unknown>
#7 0x55640f20592a <unknown>
#8 0x55640f1d3592 <unknown>
#9 0x55640f1f95e0 <unknown>
#10 0x55640f1d3363 <unknown>
#11 0x55640f19fd63 <unknown>
#12 0x55640f1a09c1 <unknown>
#13 0x55640f66ea6b <unknown>
#14 0x55640f672951 <unknown>
#15 0x55640f656b62 <unknown>
#16 0x55640f6734c4 <unknown>
#17 0x55640f63b13f <unknown>
#18 0x55640f6976f8 <unknown>
#19 0x55640f6978d6 <unknown>
#20 0x55640f6a85a6 <unknown>
#21 0x7f253789caa4 <unknown>
#22 0x7f2537929c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/BFV1_attempt_2.png'